assistFunctions.c

```diff
@@ -0,0 +1,76 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include "omp.h"
+
+int sum(int arr[], int N)
+{
+    int sum = 0; // initialize sum
+
+    // Iterate through all elements
+    // and add them to sum
+    for (int i = 0; i < N; i++)
+        sum += arr[i];
+
+    return sum;
+}
+
+// Calculates the conflict scores of a chess board
+int computeConflictScore(int N, int board[N])
+{
+    int i = 0, j = 1; // Double iterators
+    int(*conflictQueens) = malloc(sizeof(int[N]));
+
+    // Set all values in conflict queen to -1
+    for (; i < N * N; i++)
```

Comment:
why until i is `N * N` when conflictQueens is only of size `N`?

Reply:
N is only 1 dimension of the array, so if N = 8, our board would really be N * N = 64

Reply:
Also I fudged the allocation of that array, it's fixed now 😄